src/Salt/UserBundle/Controller/SignupController.php

```diff
@@ -66,12 +66,13 @@ public function signupAction(Request $request)
                 $this->sendCommand($command);
 
                 // check to see if the env var is set to "true" mailer
-                if (getenv('USE_MAIL_FEATURE') == 'always-active') {
-                  $fromEmail = getenv('MAIL_FEATURE_FROM_EMAIL');
+                if ($this->getParameter('USE_MAIL_FEATURE') == 'always-active') {
```

Comment:
This should be checking if the feature is enabled, not looking at the environment variable directly.

Ref: [Toggle Bundle example usage](https://github.com/qandidate-labs/qandidate-toggle-bundle#example-usage)